scripts/ovpsim_log_to_trace_csv.py

```diff
@@ -361,27 +378,32 @@ def main():
   parser = argparse.ArgumentParser()
   parser.add_argument("--log", type=str, help="Input ovpsim simulation log")
   parser.add_argument("--csv", type=str, help="Output trace csv_buf file")
-  parser.add_argument("-f", "--full_trace", dest="full_trace",
+  parser.add_argument("--full_trace", dest="full_trace",
                                          action="store_true",
                                          help="Generate the full trace")
-  parser.add_argument("-v", "--verbose", dest="verbose", action="store_true",
+  parser.add_argument("--verbose", dest="verbose", action="store_true",
```

Comment:
Can you keep -v to be backward compatible? You can also add -v2 for your new option

Reply:
it is only for internal testing when running standalloine so it should not affect being called from run,py or cov.py - and for larger projects such as this will become I feel cryptic names and command line arguments are not good - and we are at the beginning of the beginning of this project and so I dont feel the need for backwards compatibillity - a lot needs to change to make progress.

Reply:
I am ok to just change it for ovpsim_log_to_trace_csv.py as it won't be called by the user directly. However, I do think keep the -v is reasonable.